homeassistant/components/tractive/const.py

```diff
@@ -0,0 +1,12 @@
+"""Constants for the tractive integration."""
+
+from datetime import timedelta
+
+DOMAIN = "tractive"
+
+RECONNECT_INTERVAL = timedelta(seconds=10)
+
+TRACKER_HARDWARE_STATUS_UPDATED = "tracker_hardware_status_updated"
```

Comment:
Please prefix all the dispatch signals with the integration domain name, to make sure there's no overlap by other integrations.